integreat_cms/cms/templates/_collapsible_box.html

```diff
@@ -15,7 +16,7 @@
            class="down-arrow {% if not collapsed %}hidden{% endif %}"></i>
     </div>
     <div class="collapsible-content transition-all duration-200 ease-in-out overflow-hidden {% if collapsed %}h-0{% endif %}">
-        <div class="p-4">
+        <div class="{% if not no_padding %} p-4{% endif %}">
```

Comment:
Probably I'm being overly nit-picky here, but I think the padding on the top looks a bit large because the padding of `4` adds up to the `2` [default top margin of labels](https://github.com/digitalfabrik/integreat-cms/blob/01d2c78a3ceb4625e34e07e8b8cb110065eadf2f/integreat_cms/static/src/css/style.scss#L229)...
Could you add the class `mt-0` to all labels which are the first elements in a box? Or do you have a better idea how to fix this?

Reply:
Hm good point. So, so far we only had 1rem padding to the sides and bottom, and 0.5rem to the top (stemming from the mt-2 of the label). But on the opening hours box we had 1rem to all sides (also top). So on our production server the spacing is inconsistent. I think we started to correct this on with the collapsible boxes for pages, where we added mt-0 for labels, so that we have 1rem to all sides on every box. I guess this is the way to go for POIs and events too - thus I will follow your suggestion and add mt-0 to all labels that are the first elements in a box

Reply:
This should be solved now. Could you please check again, if it's working as expected? :)